### DISCUSSION TITLE
Number of rows visible in a page [Feb 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5156] [Discord Threads]
> 🙋‍♂️ **kannthus**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### You will have to configure environment variable `DB_QUERY_LIMIT_DEFAULT`; this defaults to 25  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1062822903067988088)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
